internal/cli/project_apply.go

```diff
@@ -514,6 +537,12 @@ func (c *ProjectApplyCommand) Flags() *flag.Sets {
 			Default: "5m",
 			Usage:   "Interval between polling to generate status reports if polling is enabled.",
 		})
+
+		f.StringVar(&flag.StringVar{
+			Name:   "ondemand-runner",
```

Comment:
I continue to think for user facing stuff, we shouldn't separate "ondemand" from "runner" and consider ondemand is just an implementation detail of how Waypoint just has "runners." Understanding both feels like some cognitive overhead. In that case, this should probably just be `runner-config` or something. We should maybe come up with a noun to describe our runner configs, is it "config" "role" "class" etc.

Reply:
Unsure how to model this, mostly because it really is specific to the concept of on-demand runners. I worry if it's just `runner-config` folks might presume it means something about the static runners? But to your point, it feels like the concept of users worrying about a static runner are going away.

Reply:
Yeah, I feel the same way too, its a bit confusing either way. We can keep it for now, the CLI flags aren't a part of any of our compat guarantees anyways and this is fairly minor.